docs/guides/src/main/server/configuration_overview.adoc

```diff
@@ -0,0 +1,196 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+<#import "/templates/link.adoc" as link>
+
+<@tmpl.guide
+title="Configuring Keycloak"
+summary="An overview of the server configuration">
+
+In this guide, you will learn the core concepts around the server configuration and how to configure the server using the different
+configuration options available. You will also learn how to properly configure the server to achieve an optimal runtime for faster
+startup and low memory footprint.
+
+== Configuration Options
+
+All the available configuration options can be found by either running the `help` command for individual commands:
+
+<@kc.all parameters="--help"/>
+
+Or by looking at all the configuration options at the <@link.guide title="All configuration"/> section.
+
+These options can be set using different formats:
+
+.Command-line argument
+<@kc.all parameters="--<category>-<sub-category>-<property>=<value>"/>
+
+.Environment variable
+```
+export KC_CATEGORY_SUB_CATEGORY_PROPERTY=<value>
+```
+
+.Properties at `conf/keycloak.properties`
+```
+<category>.<sub-category>.<property>=<value>
+```
+
+The format you should use is use-case specific but most of the time it should be enough to use the CLI. As an example, when running
+the server as a container you should be fine using both environment variables or command-line options.
+
+[NOTE]
+If you are unsure about the format of a specific property, consider looking at the <@link.guide title="All configuration"/> section.
+
+If the same configuration option is defined in different configuration sources, the effective value will be resolved by respecting the following order:
```

Comment:
change "will be" to "is:
in most cases, present tense works. It sounds more active, but means the same thing.